Harris Pharmacy/HarrisPharamacy/HarrisPharamacy.App/Pages/AppointmentInformation.razor

```diff
@@ -72,7 +76,9 @@
 
     protected override async Task OnInitializedAsync()
     {
-        appt = await AppointmentService.GetAppointmentAsync(patientListID);
+        //appt = await AppointmentService.GetAppointmentAsync(patientListID);
```

Comment:
If you don't need these commented lines, maybe remove them to clean up the code?